en/SettingsView/General.md

```diff
@@ -75,7 +75,25 @@ Settings include:
 - **Libre Pilot**: Autoconnect to Libre Pilot autopilot
 - **UDP**: Autoconnect to UDP
 - **RTK GPS**: Autoconnect to RTK GPS device
-- **NMEA GPS Device / Baudrate**: If detected, GPS information will be used for ground station location and follow me support.
+
+## Ground Station GPS (NMEA GPS Device) {#nmea_gps}
+
+The Ground Station GPS section is used to choose the source of GPS data which will be used for the ground control station location. When there is a valid position provided by the NMEA GPS device, the location of the ground control station will be displayed on the map with a purple `Q` icon. If the GPS provides a heading, it will be indicated by the `Q` icon. The GCS location is also used to provide follow me support ([follow me](https://docs.px4.io/en/flight_modes/follow_me.html) is currently implemented for PX4 only).
+
```

Comment:
Here, add a new paragraph (just an idea, if you want to improve please do so ...)

> You can change the GPS source using the drop down menu in the *NMEA GPS Device* field.  The configuration options will change based on the selected device (as shown below).

Reply:
I like it. Updated.